code/Makefile

```diff
@@ -0,0 +1,12 @@
+# Default Target
+.PHONY: all
+all: CAVITY_2D_STAGGERED
+
+# CAVITY_2D_STAGGERED
+CAVITY_2D_STAGGERED: clean
+	gfortran -o CAVITY_2D_STAGGERED CAVITY_2D_STAGGERED.f90
```

Comment:
The file in directory ends with “F”90, but you name it “f”90 here.  It won’t work under a case sensitive file system like what Linux uses.